data/Santini_2016/metadata.yml

```diff
@@ -0,0 +1,349 @@
+    ORCID: 0000-0002-8319-8862
+    additional_role: contact
+  - last_name: Catherine
+    given_name: Lestrange
+    affiliation: University of Technology, Sydney
+    ORCID: .na
+  austraits_curators: Gillian Kowalick, Elizabeth Wenk
+dataset:
+  data_is_long_format: no
+  custom_R_code:           '
+    data %>%
+      mutate(stem_vessel_diameter = (`Vessel diameter a (u)`+`Vessel diameter b (u)`)/2)
+  '
+  collection_date: 2014-07/2014-09
+  taxon_name: Species
+  location_name: site
+  description: This study examined variation in branch xylem anatomy and foliar carbon
+    stable isotopes (d13C) as a measure of water-use efficiency (WUE) in seven co-occurring
+    species, Acacia aneura, Acacia bivenosa, Corymbia opaca, Eucalyptus camaldulensis,
+    Erythrina vespertilio, Hakea sp., and Psydrax latifolia, in an arid zone open
+    Corymbia savanna on the Ti Tree Basin, Northern Territory, Australia.
+  basis_of_record: field
+  life_stage: adult
+  sampling_strategy: The study site was located in the Ti Tree Basin (22.7048.5600S,
+    133.24057.6700E), approximately 180 km north of Alice Springs, Northern Territory,
+    Australia. The site is arid with a mean annual rainfall of 347 mm year-1 (Australian
+    Bureau of Meteorology 2014, BoM station 15507), most of which falls in monsoonal
+    summer storms. Mean minimum and maximum annual temperatures are 15 and 31 degrees
+    celcius, respectively. The site reaches maximum air temperatures of 40 degrees
+    celcius in the summer months (December-February).The Ti Tree Basin extends over
+    5500 km2 and contains the ephemeral Hanson and Woodforde rivers, which only flow
+    after intense storms (Harrington et al. 2002). During July-September 2014, we
+    collected upper canopy terminal branches of seven co-occurring woody species from
+    the Ti Tree Basin Acacia aneura, Acacia bivenosa, Corymbia opaca, Eucalyptus camaldulensis,
+    Erythrina vespertilio, Hakea sp., and Psydrax latifolia. Branches of different
+    species varied in diameter (2-7 mm), but we standardised our collection by sampling
+    the terminal 25 cm of each branch.
+  original_file: AusTRAITS_Santini (2).xls and AusTRAITS_Santini (3).xls in raw data
+    folder
+  notes: none
+locations:
+  Ti-Tree Basin:
+    latitude (deg): -22.1302
+    longitude (deg): 133.416
+    description: Arid zone open Corymbia savanna on the Ti Tree Basin, Northern Territory,
+      Australia.
+    precipitation, MAP (mm): 347
+contexts:
+- context_property: tissue sampled
+  category: method
+  var_in: method_context
+  values:
+  - value: sapwood
+    description: Measurement made on sapwood.
+  - value: total wood
+    description: Measurement made on complete wood disc.
+traits:
+- var_in: stem_vessel_diameter
+  unit_in: um
+  trait_name: stem_vessel_diameter
+  entity_type: individual
+  value_type: raw
+  basis_of_value: measurement
+  replicates: 1
```

Comment:
The mean I took is for the two dimensions overall, so that is still just 1 diameter. However, you're right that this is an individual mean but I haven't found anywhere reporting the number samples, so I'll change to individual mean with unknown replicate counts